snowfakery/output_streams.py

```diff
@@ -216,20 +224,20 @@ def __init__(self, file):
 
     def write_single_row(self, tablename: str, row: Dict) -> None:
         if self.first_row:
-            self.stream.write("[")
+            self.write("[")
             self.first_row = False
         else:
-            self.stream.write(",\n")
+            self.write(",\n")
         values = {"_table": tablename, **row}
-        self.stream.write(json.dumps(values))
+        self.write(json.dumps(values))
 
     def close(self) -> Optional[Sequence[str]]:
         if not self.first_row:
-            self.stream.write("]\n")
-        return super().close()
+            self.write("]\n")
+        super().close()
```

Comment:
Shouldn't this return the return value from the super call?

Reply:
38c9eacf